src/main/java/uk/gov/di/ipv/cri/common/library/service/ConfigurationService.java

```diff
@@ -37,33 +41,52 @@ public enum SSMParameterName {
 
     @ExcludeFromGeneratedCoverageReport
     public ConfigurationService() {
-        this.clock = Clock.systemUTC();
-        this.ssmProvider = ParamManager.getSsmProvider();
-        this.secretsProvider = ParamManager.getSecretsProvider();
-        this.parameterPrefix =
-                Objects.requireNonNull(
-                        System.getenv("AWS_STACK_NAME"), "env var AWS_STACK_NAME required");
+        this(
+                ParamManager.getSsmProvider(
+                        SsmClient.builder().httpClient(UrlConnectionHttpClient.create()).build()),
+                ParamManager.getSecretsProvider(
```

Comment:
please also test if the following amend works in your stack:
```
        ParamManager.getSecretsProvider(
                SecretsManagerClient.builder()
                        .credentialsProvider(EnvironmentVariableCredentialsProvider.create())
                        .httpClient(UrlConnectionHttpClient.create())
                        .region(Region.of(System.getenv("AWS_REGION")))
                        .build()),
```

Reply:
same as above, I can add these in am pretty sure they work only removed it this morning.